setup.py

```diff
@@ -56,7 +67,7 @@ def get_version():
             "Programming Language :: Python :: 3.7",
         ],
         platforms="any",
-        packages=["pyglmnet"],
+        packages=package_tree("pyglmnet"),
```

Comment:
why not:

```py
from setuptools import find_packages
```

and then here:

```suggestion
        packages=find_packages(),
```

Reply:
I was gonna do that, not sure which way is good. Will come fix it after lunch!